Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -306,17 +306,19 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
 
     /// Mutes the channel this controller manages.
     ///
-    /// - Parameter completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
-    ///                         If request fails, the completion will be called with an error.
+    /// - Parameters:
+    ///   - expiration: Duration of mute in milliseconds.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
+    ///                 If request fails, the completion will be called with an error.
     ///
-    public func muteChannel(completion: ((Error?) -> Void)? = nil) {
+    public func muteChannel(expiration: Int? = nil, completion: ((Error?) -> Void)? = nil) {
```

Comment:
All our expirations are in seconds (TimeInterval). I think we really should try to keep it consitant with the rest of the SDK as well as adhering to iOS guidelines. Example, Slow Mode, Pin Expiration, are all in seconds and use TimeInterval as the type

Reply:
Actually, in slow mode, the cooldown is an `Int` as well, but it is seconds. 😞 Ideally, all time-related stuff should be `TimeInterval` which represents seconds.

Reply:
The problem is that according to our API, both [Pin Expiration](https://getstream.io/chat/docs/react/pinned_messages/?language=javascript) and [Slow Mode](https://getstream.io/chat/docs/react/slow_mode/?language=javascript&q=slow%20mode) use `seconds` when [Mute Expiration](https://getstream.io/chat/docs/react/muting_channels/?language=javascript) uses `milliseconds`. I agree that it's not convenient, but I still think that on this, we should align with our backend and other SDKs that leverage milliseconds in this case.

Reply:
I can see that Android also uses milliseconds. Then lets keep it like this 👍